Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.h

```diff
@@ -180,26 +180,24 @@ class GStreamerRegistryScanner {
     void fillAudioRtpCapabilities(Configuration, RTCRtpCapabilities&);
     void fillVideoRtpCapabilities(Configuration, RTCRtpCapabilities&);
 
-#define WEBRTC_EXPERIMENTS_HDREXT "http://www.webrtc.org/experiments/rtp-hdrext/"
-    Vector<const char*> m_commonRtpExtensions {
-        "http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01",
-        WEBRTC_EXPERIMENTS_HDREXT "abs-send-time",
-        GST_RTP_HDREXT_BASE "sdes:mid",
-        GST_RTP_HDREXT_BASE "sdes:repaired-rtp-stream-id",
-        GST_RTP_HDREXT_BASE "sdes:rtp-stream-id",
-        GST_RTP_HDREXT_BASE "toffset"
```

Comment:
Why are you removing these contants here and below?

Reply:
Ah you mean the defines? I had issues with that and the `_s` operator.

Reply:
Nevermind, reverted this change.